### PR TITLE
Run examples on universal setup

### PR DESCRIPTION
**Pull request by moret**
This PR allows running the examples on a hapi server to make it easier to detect when React warns about not being able to reuse markup.

It also fixes the first warning found, related to using div's inside p's. The JSX was:
````
<p><div>...</div></p>
````
Resulting in the browser's DOM:
````
<p></p><div>...</div><p></p>
````
I changed it to avoid differences between the server rendering and the browser DOM.

The remaining React warning is related to #201 - Prefixing breaks server rendering.

### CODE REVIEW

File: examples/app.jsx [hunk 1]
```diff
@@ -94,36 +94,29 @@ var App = React.createClass({
 
     return (
       <div>
-        <p><HoverMessage /></p>
```
Comment (ianobermiller) [inline examples/app.jsx]: Why change all these? `<p>` as a container tag is much nicer.

**Comment by ianobermiller** [comment]
Can you sync with @jurgob, who submitted something very similar in #315? We'll only be able to merge one of them.

File: examples/app.jsx (hunk 1 above)
Comment (ianobermiller) [inline examples/app.jsx]: Sorry, I just read the PR description. You're right, and that is crazy. I guess this is why we use `div`s for everything.
image

**Comment by moret** [comment]
Yeah, I liked the container `<p>` too. 8(

I didn't notice there was another PR on the same subject. I will check it out with @jurgob .

**Comment by ianobermiller** [comment]
thanks @moret!

**Pull request closed without merging**